
dlio:
  use_loaded_state: true
  state:
    quat:
      w: 1
      x: 0
      y: 0
      z: 0

    pose:
      x: 0
      y: 0
      z: 0
  
  map_link: "/home/nakul/Desktop/vectr/ws/src/map_1.pcd"